sql/migrations/20170113181857_addConstraintsToPeers.sql

```diff
@@ -0,0 +1,11 @@
+/* Add constraints to improve upserts
+ *
+ */
+
+BEGIN;
+
+ALTER TABLE peers
+  ADD CONSTRAINT unique_address
+UNIQUE (ip, port);
```

Comment:
We already have unique index on those fields, so `USING INDEX peers_unique;`
Maintaining both unique index and unique constraint is costly and we don't need it.